drivers/gpio/gpio-bcm-kona.c

```diff
@@ -597,7 +597,6 @@ static int bcm_kona_gpio_probe(struct platform_device *pdev)
 		return -ENOMEM;
 
 	kona_gpio->pdev = pdev;
-	platform_set_drvdata(pdev, kona_gpio);
```

Comment:
bad news for this commit;
we'll need to change the description a bit;
`platform_set_drvdata()` was added from the beginning, but never used;

it looks like it was a copy+paste error; someone copied the driver, and just did not remove it;

commit description should be more llike 
```
The platform_set_drvdata() call was never used, ever since the driver was
originally added (back in 2013).
It looks like this was a copy+paste line of code which was taken from
another driver.
```